subprojects/dependency-management/src/integTest/resources/org/gradle/integtests/ivy/IvyMappingToVariantIntegrationTest/ivyRepo/foo/my-core/1.0.0/my-core-1.0.0-ivy.xml

```diff
@@ -0,0 +1,24 @@
+<ivy-module xmlns:e="http://ant.apache.org/ivy/extra" version="2.0">
```

Comment:
This file and all others in `ivyRepo` really should go away. If the test remains required, please leverage the existing test fixtures to create them as part of the test.

Reply:
These files are gone!